boards/sim/sim/sim/README.txt

```diff
@@ -1158,15 +1158,12 @@ rpserver
     4>Rpmsg UsrSock:
       Start the usrsock server on rpserver:
```

Comment:
This comment also became outdated. Could be something like: confirm that "rptun proxy" kernel thread is running.

Reply:
Done.